indico/modules/events/surveys/templates/display/survey_questionnaire.html

```diff
@@ -33,7 +33,17 @@
         {% endcall %}
     {% endfor %}
     {% call form_footer(form) %}
-        <input class="i-button big highlight" type="submit" value="{% trans %}Save answers{% endtrans %}">
+        {% if partial_completion %}
+            <a class="i-button big js-save-answers"
+               data-href="{{ url_for('.display_save_answers', survey) }}"
+               data-params-selector="#survey-questionnaire input[name^=question_]:checked,
+                                     #survey-questionnaire [name^=question_]:not(:radio):not(:checkbox)"
+               data-method="POST"
```

Comment:
I'd move this one up (so method is right below href). I think we do it like that everywhere else